leptos_reactive/src/signal.rs

```diff
@@ -1997,13 +1996,14 @@ impl NodeId {
                 None
             };
 
-            // mark descendants dirty
-            runtime.mark_dirty(*self);
-
             // notify subscribers
-            if updated.is_some() && !runtime.batching.get() {
```

Comment:
Yeah looks like this batching check is redundant with the one that's already in `run_effects`, so this makes sense.